src/vs/editor/common/standalone/standaloneEnums.ts

```diff
@@ -246,86 +246,87 @@ export enum EditorOption {
 	linkedEditing = 70,
 	links = 71,
 	matchBrackets = 72,
-	minimap = 73,
-	mouseStyle = 74,
-	mouseWheelScrollSensitivity = 75,
-	mouseWheelZoom = 76,
-	multiCursorMergeOverlapping = 77,
-	multiCursorModifier = 78,
-	multiCursorPaste = 79,
-	multiCursorLimit = 80,
-	occurrencesHighlight = 81,
-	overviewRulerBorder = 82,
-	overviewRulerLanes = 83,
-	padding = 84,
-	pasteAs = 85,
-	parameterHints = 86,
-	peekWidgetDefaultFocus = 87,
-	placeholder = 88,
-	definitionLinkOpensInPeek = 89,
-	quickSuggestions = 90,
-	quickSuggestionsDelay = 91,
-	readOnly = 92,
-	readOnlyMessage = 93,
-	renameOnType = 94,
-	renderControlCharacters = 95,
-	renderFinalNewline = 96,
-	renderLineHighlight = 97,
-	renderLineHighlightOnlyWhenFocus = 98,
-	renderValidationDecorations = 99,
-	renderWhitespace = 100,
-	revealHorizontalRightPadding = 101,
-	roundedSelection = 102,
-	rulers = 103,
-	scrollbar = 104,
-	scrollBeyondLastColumn = 105,
-	scrollBeyondLastLine = 106,
-	scrollPredominantAxis = 107,
-	selectionClipboard = 108,
-	selectionHighlight = 109,
-	selectOnLineNumbers = 110,
-	showFoldingControls = 111,
-	showUnused = 112,
-	snippetSuggestions = 113,
-	smartSelect = 114,
-	smoothScrolling = 115,
-	stickyScroll = 116,
-	stickyTabStops = 117,
-	stopRenderingLineAfter = 118,
-	suggest = 119,
-	suggestFontSize = 120,
-	suggestLineHeight = 121,
-	suggestOnTriggerCharacters = 122,
-	suggestSelection = 123,
-	tabCompletion = 124,
-	tabIndex = 125,
-	unicodeHighlighting = 126,
-	unusualLineTerminators = 127,
-	useShadowDOM = 128,
-	useTabStops = 129,
-	wordBreak = 130,
-	wordSegmenterLocales = 131,
-	wordSeparators = 132,
-	wordWrap = 133,
-	wordWrapBreakAfterCharacters = 134,
-	wordWrapBreakBeforeCharacters = 135,
-	wordWrapColumn = 136,
-	wordWrapOverride1 = 137,
-	wordWrapOverride2 = 138,
-	wrappingIndent = 139,
-	wrappingStrategy = 140,
-	showDeprecated = 141,
-	inlayHints = 142,
-	editorClassName = 143,
-	pixelRatio = 144,
-	tabFocusMode = 145,
-	layoutInfo = 146,
-	wrappingInfo = 147,
-	defaultColorDecorators = 148,
-	colorDecoratorsActivatedOn = 149,
-	inlineCompletionsAccessibilityVerbose = 150,
-	quickSuggestionsMinimumLength = 151,
-	tabSuggest = 152
+	actionBar = 73,
```

Comment:
Could you comment on the intent to change all of these enums?

Reply:
No, it wasn't unexpected. `standaloneEnums.ts` is a generated file. In this case, it was regenerated when the `actionBar` was added to the `export const enum EditorOption` in `src/vs/editor/common/config/editorOptions.ts`.

These numbers are not "well known". They are never used outside the enumeration in which they get generated. So, it doesn't matter that we added `actionBar` near `minimap`, which was done intentionally.